api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -690,7 +690,7 @@ public void testCreateWorkspace() {
 
   @Test
   public void testCreateWorkspace_resetBillingAccountOnFailedSave() {
-    doThrow(RuntimeException.class).when(workspaceDao).save(any(DbWorkspace.class));
+    doThrow(new RuntimeException()).when(workspaceDao).save(any(DbWorkspace.class));
```

Comment:
Would fail locally but not CI if not updated to this

Reply:
interesting